storybook/styles/docs.css

```diff
@@ -332,9 +332,11 @@
 
 /* ELEMENTS | BLOCKQUOTE */
 .sbdocs.sbdocs-blockquote {
```

Comment:
Is the idea here to still support blockquote? If so, you can't really render any since the markdown will defer to the Highlight component. Therefore, I would remove the CSS.

Reply:
We are not planning to support it as quoting someone is not in the projects ;) I'll remove it. If we ever need to use a blockquote, which we doubt, this might change for now removing.